plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
I was just wondering if something like
```
#include <ntverp.h>
#if VER_PRODUCTBUILD > 7600
```
couldn't be more "orthodox", considering that's adding an *useless* variable to DX SDK builds.